### DISCUSSION TITLE
sqlx_logging_level not work?

### DISCUSSION THREAD

**Discussion by traitmeta** [SeaQL/sea-orm#1933] [Q&A]
### I use Postgres and the connection setting is this.
```
opt.max_connections(100)
        .min_connections(5)
        .connect_timeout(Duration::from_secs(8))
        .acquire_timeout(Duration::from_secs(8))
        .idle_timeout(Duration::from_secs(8))
        .max_lifetime(Duration::from_secs(8))
        .sqlx_logging(true)
        .sqlx_logging_level(log_level)
        .set_schema_search_path("public"); 
```
### If the log_level is info and log is Info, and if the log_level is Error and log is Error.

<img width="717" alt="image" src="https://github.com/SeaQL/sea-orm/assets/12974811/d255f078-7b4e-41aa-b2f5-38f69f05d2a9">
<img width="737" alt="image" src="https://github.com/SeaQL/sea-orm/assets/12974811/52fb2322-bd4e-4dd4-aca2-095757424fc2">

**So, what's the function with `sqlx_logging_level`?**

**Comment by tyt2y3** [comment]
It's a wrapper over https://docs.rs/sqlx/latest/sqlx/trait.ConnectOptions.html#tymethod.log_statements
